controllers/nodehealthcheck_controller.go

```diff
@@ -325,8 +326,7 @@ func (r *NodeHealthCheckReconciler) markHealthy(node *v1.Node, nhc *remediationv
 	return nil
 }
 
-func isHealthy(conditionTests []remediationv1alpha1.UnhealthyCondition, nodeConditions []v1.NodeCondition) bool {
-	now := time.Now()
+func isHealthy(conditionTests []remediationv1alpha1.UnhealthyCondition, nodeConditions []v1.NodeCondition, now time.Time) bool {
```

Comment:
NIT: consider converting this to a method, that way we will not need to pass it `now` variable.

Reply:
done